lua/cmp_cmdline/init.lua

```diff
@@ -19,10 +19,25 @@ local MODIFIER_REGEX = {
   vim.regex([=[verb\%[ose]\s*]=]),
   vim.regex([=[vert\%[ical]\s*]=]),
 }
+
+-- Matches (examples):
+-- 23
+-- -
+-- +23
+-- .
+-- $
+local line_address_regex = [=[\%(\$\|\.\|[-+]\d*\|\d\+\)\?]=]
+
 local COUNT_RANGE_REGEX = {
-  vim.regex([=[\%(\d\+\|\$\),\%(\d\+\|\$\)\s*]=]),
   vim.regex([=['<,'>\s*]=]),
-  vim.regex([=[\%(\d\+\|\$\)\s*]=]),
+  -- The regex below should match these examples:
+  -- 4
+  -- 5,
+  -- 6,7
+  -- -6,+7
+  -- +6,$
+  -- -6,-3t.
+  vim.regex(line_address_regex .. [=[,\?]=] .. line_address_regex .. [=[\s*[tmyd]\?\s*]=] .. line_address_regex),
```

Comment:
Thank you for contribution!

Sirry. I can't understand the `tmyd`...

Reply:
Not entirely sure but these are shorthands for copy/t(`:h co`), move(m), yank, and delete that @artemave is trying to match in this regex since they are commonly used to manipulate lines in an Ex command.

Reply:
> Not entirely sure but these are shorthands for copy/t(`:h co`), move(m), yank, and delete that @artemave is trying to match in this regex since they are commonly used to manipulate lines in an Ex command.

Yes, that. Probably not an exhaustive list.

Reply:
I think we can use `\%[]` pattern for it?

For example, `1,` is partial pattern but it should be matched.

Reply:
> For example, 1, is a partial pattern but it should be matched.

It's matched, as far as I can see in manual tests.